jobqueue/scheduler/lsf.go

```diff
@@ -410,6 +414,13 @@ func (s *lsf) initialize(config interface{}, logger log15.Logger) error {
 	//*** we probably don't need this if we won't be having a global max
 	// specified by the user
 
+	// if a job becomes lost, scheduler needs to ssh to the host to check on the
+	// process, so we store our private key, currently taken from hardcoded
+	// path
+	if content, err := os.ReadFile(internal.TildaToHome("~/.ssh/id_rsa")); err == nil {
```

Comment:
Hi,
I'm curious about this. Does it preclude using ECDSA or Ed25519 keys ?

Reply:
Hi, this is a work-in-progress PR; I'm going to remove the hard-coding and make it a config option. Would that address your concern?

Reply:
For instance, yes 👍 Thanks !